sdk/examples/module/find.ts

```diff
@@ -0,0 +1,5 @@
+import VeraxSdk from "../../src/VeraxSdk";
```

Comment:
should be removed (only `findBy` and `findOneBy` exist in read)

Reply:
Don't we want an example to find a `Module`?

Reply:
Moved to `UtilsDataMapper`